yaml/landuse.yaml

```diff
@@ -797,6 +802,14 @@ filters:
     output:
       <<: *output_properties
       kind: {col: landuse}
+      tz_drop_polygon:
```

Comment:
Let's just keep `mz` namespace, otherwise it becomes confusing with both? We've renamed it to "metazen" in WOF ;)